.github/workflows/crds.yaml

```diff
@@ -0,0 +1,56 @@
+name: Update CRDs for chart repo
+
+on:
+  release:
+    types: [published]
```

Comment:
Should we set filter to avoid RC tags?

Reply:
We could, but i haven't seen any rc tag, should we filter out with `-rc` suffix?

Reply:
Hm... it's weird, i definitely remember that @f41gh7 sometimes releases RC versions to test them in our environments. Maybe he deletes them afterwards.

Reply:
https://hub.docker.com/r/victoriametrics/operator/tags?page=&page_size=&ordering=&name=rc